.github/workflows/deb-package.yml

```diff
@@ -0,0 +1,268 @@
+name: Deb-Package
+
+env:
+  PKG_PREFIX: boinc
+  ARCH: amd64
+  MANTAINER: BOINC <***@***.com>
+  HOMEPAGE: https://boinc.berkeley.edu/
+  DESCRIPTION: BOINC lets you help cutting-edge science research using your computer. The BOINC app, running on your computer, downloads scientific computing jobs and runs them invisibly in the background. It's easy and safe.
+  BASEREPO: https://boinc.berkeley.edu/dl/linux # no trailing slash
+  GH_REPO_API: BOINC/boinc # no trailing or prefix slash
+  PUBKEY: boinc-202305.gpg # keep extension
+
+on:
+  workflow_dispatch:
+    inputs:
+      package_type:
+        description: 'package type to generate'
+        type: choice
+        options:
+          - linux_client-vcpkg
+          - linux_manager-without-webview
```

Comment:
would be netter to use this one: `linux_manager-with-webview-vcpkg`
This will make this flow consistent

Reply:
Thats the name with which the artifacts are saved, in a following step the suffix '-vcpkg' is removed.

If you feel its better to have both the output names with the suffix i can do that.

Reply:
I just need to be sure that we're packing the correct artifacts. I don't really care about the name